<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>456</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>nanoGG</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/res/available.png</normaloff>:/res/available.png</iconset>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <property name="spacing">
     <number>2</number>
    </property>
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabs">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tabMain">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <attribute name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/res/unavailable.png</normaloff>:/res/unavailable.png</iconset>
       </attribute>
       <attribute name="title">
        <string>nanoGG</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::ExpandingFieldsGrow</enum>
        </property>
        <property name="rowWrapPolicy">
         <enum>QFormLayout::DontWrapRows</enum>
        </property>
        <property name="labelAlignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="formAlignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="horizontalSpacing">
         <number>5</number>
        </property>
        <property name="verticalSpacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>7</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <item row="0" column="1">
         <widget class="QLineEdit" name="ggNumber">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::NoContextMenu</enum>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="maxLength">
           <number>32767</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelHaslo">
          <property name="text">
           <string>Hasło:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="ggPassword">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelOpis">
          <property name="text">
           <string>Opis:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPlainTextEdit" name="ggDescription">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="pushButtonUstawOpis">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>48</height>
           </size>
          </property>
          <property name="text">
           <string>Ustaw</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelGGNumber">
          <property name="text">
           <string>#GG:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>598</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuNanoGG">
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>nanoGG</string>
    </property>
    <addaction name="actionZakoncz"/>
    <addaction name="actionImport_listy"/>
    <addaction name="actionPobierz_avatary"/>
    <addaction name="separator"/>
    <addaction name="actionUstawienia_domyslne"/>
    <addaction name="separator"/>
    <addaction name="actionO_programie"/>
   </widget>
   <widget class="QMenu" name="menuOkna">
    <property name="title">
     <string>Okna</string>
    </property>
    <addaction name="actionKontakty"/>
    <addaction name="actionZamknijKarte"/>
   </widget>
   <widget class="QMenu" name="menuStatus">
    <property name="title">
     <string>Status</string>
    </property>
    <addaction name="actionPorozmawiajmy"/>
    <addaction name="actionDostepny"/>
    <addaction name="actionJestem_zajety"/>
    <addaction name="actionZaraz_wracam"/>
    <addaction name="actionNiewidoczny"/>
    <addaction name="actionNiedostepny"/>
    <addaction name="separator"/>
    <addaction name="actionTylko_dla_znajomych"/>
   </widget>
   <addaction name="menuNanoGG"/>
   <addaction name="menuOkna"/>
   <addaction name="menuStatus"/>
  </widget>
  <action name="actionZakoncz">
   <property name="text">
    <string>Zakończ nanoGG</string>
   </property>
  </action>
  <action name="actionDostepny">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/available.png</normaloff>:/res/available.png</iconset>
   </property>
   <property name="text">
    <string>Dostępny</string>
   </property>
  </action>
  <action name="actionPorozmawiajmy">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/chat.png</normaloff>:/res/chat.png</iconset>
   </property>
   <property name="text">
    <string>Porozmawiajmy</string>
   </property>
  </action>
  <action name="actionJestem_zajety">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/dnd.png</normaloff>:/res/dnd.png</iconset>
   </property>
   <property name="text">
    <string>Jestem zajęty</string>
   </property>
  </action>
  <action name="actionZaraz_wracam">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/away.png</normaloff>:/res/away.png</iconset>
   </property>
   <property name="text">
    <string>Zaraz wracam</string>
   </property>
  </action>
  <action name="actionNiewidoczny">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/invisible.png</normaloff>:/res/invisible.png</iconset>
   </property>
   <property name="text">
    <string>Niewidoczny</string>
   </property>
  </action>
  <action name="actionNiedostepny">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/unavailable.png</normaloff>:/res/unavailable.png</iconset>
   </property>
   <property name="text">
    <string>Niedostępny</string>
   </property>
  </action>
  <action name="actionTylko_dla_znajomych">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tylko dla znajomych</string>
   </property>
   <property name="iconText">
    <string>Tylko dla znajomych</string>
   </property>
   <property name="toolTip">
    <string>Tylko dla znajomych</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionZamknijKarte">
   <property name="text">
    <string>Zamknij kartę</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionImport_listy">
   <property name="text">
    <string>Import Listy</string>
   </property>
  </action>
  <action name="actionO_programie">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/available.png</normaloff>:/res/available.png</iconset>
   </property>
   <property name="text">
    <string>O programie</string>
   </property>
  </action>
  <action name="actionKontakty">
   <property name="text">
    <string>Kontakty</string>
   </property>
  </action>
  <action name="actionUstawienia_domyslne">
   <property name="text">
    <string>Ustawienia domyślne</string>
   </property>
  </action>
  <action name="actionPobierz_avatary">
   <property name="text">
    <string>Pobierz avatary</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionZakoncz</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
